src/cpp/include/openvino/genai/speculative_decoding/perf_metrics.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "openvino/genai/perf_metrics.hpp"
+#include "openvino/genai/visibility.hpp"
+
+
+namespace ov::genai {
+
+struct OPENVINO_GENAI_EXPORTS SDPerfMetrics : public ov::genai::PerfMetrics {
+    ov::genai::MeanStdPair ttst;  // Time to the second token (in ms).
+    ov::genai::MeanStdPair ttl;  // latency from the third token (in ms).
+    size_t num_accepted_tokens; // num tokens, which was accepted but main model
+    size_t num_iterations; // num inference
+
+    ov::genai::MeanStdPair get_ttst();
+    ov::genai::MeanStdPair get_latency();
+
+    SDPerfMetrics() = default;
```

Comment:
```suggestion
```
What is it for? `PerfMetrics` doesn't have it. Same for `SDPerModelsPerfMetrics()`: you can specify the defaults inplace

Reply:
removed